src/lib.rs

```diff
@@ -172,7 +175,7 @@ pub use crate::file::{
 };
 pub use crate::spooled::{spooled_tempfile, SpooledData, SpooledTempFile};
 
-/// Create a new temporary file or directory with custom parameters.
+/// Create a new temporary file or directory with custom parameters and permissions.
```

Comment:
IMO, there's no need to elaborate here. Parameters imply permissions, etc. (could also just call it "options" if you find that less confusing).